Resources/public/js/components/news/list/main.js

```diff
@@ -0,0 +1,10 @@
+define(function() {
+
+    'use strict';
```

Comment:
for more information about strict mode see https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Strict_mode